src/components/verticaltabs/_verticaltabs.scss

```diff
@@ -0,0 +1,55 @@
+.cmp-verticaltabs{
+    font-size: $font-m;
+    margin: $field-margin;
+    position: relative;
+  
+  &__tabs-container{
+    display: flex;
+    margin-top: 16px;
+  }
+  &__tabpanel{
+    flex: 1;
+    padding: 16px 56px;
+  }
+  &__tabpanel--active{
+    
+  }
+  &__tablist{
+    width: 150px;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+  &__tab{
+    padding: 12px 4px 12px 12px;
+    margin-top: 8px;
+  }
+  &__tab:hover{
+    border-left: 2px solid $light-gray;
+  }
+  &__tab--active {
+    border-left: 2px solid;
+  }
+  &__longdescription {
```

Comment:
use description mixin for description, short description and long description

Reply:
done